mkdocs.yml

```diff
@@ -65,8 +65,7 @@ theme:
 
 plugins:
   - open-in-new-tab
-  - macros:
-      include_dir: .
+  - macros
```

Comment:
Removed because it is redundant.
The default is to include the current folder `.`.